data/sql_updates/create_committee_history.sql

```diff
@@ -79,35 +67,33 @@ select distinct on (dcp.cmte_sk, cycle)
     dcp.cmte_custodian_title as custodian_name_title,
     dcp.cmte_custodian_zip as custodian_zip,
     -- Properties
-    dcp.cmte_email as email,
+    fec_yr.cmte_email as email,
     dcp.cmte_fax as fax,
-    dcp.cmte_web_url as website,
+    fec_yr.cmte_url as website,
     dcp.form_tp as form_type,
     dcp.leadership_pac as leadership_pac,
-    dcp.load_date as load_date,
```

Comment:
If we have to get rid of `load_date` from dimcmteproperties, we can add `date_entered` and `date_changed`.

@jwchumley and @paulclark2 do you think that `date_entered` is comparable enough to `load_date` that we could call it that?